front_end/content.py

```diff
@@ -248,6 +261,39 @@ def get_problem_details(course, assignment, problem, format_content=False, parse
 
     return problem_dict
 
+def get_dict_from_log(file_path, year="No filter", month="No filter", day="No filter"):
```

Comment:
Is this function being used for anything other than log summarization? If not, please move it to summarize_logs.py. If so, please give it a more descriptive name.